Modules/xSharePoint/DSCResources/MSFT_xSPOutgoingEmailSettings/MSFT_xSPOutgoingEmailSettings.psm1

```diff
@@ -0,0 +1,86 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $WebAppUrl,
+        [parameter(Mandatory = $true)] [System.String] $SMTPServer,
+        [parameter(Mandatory = $true)] [System.String] $FromAddress,
+        [parameter(Mandatory = $true)] [System.String] $ReplyToAddress,
+        [parameter(Mandatory = $true)] [System.String] $CharacterSet
+        
+    )
+
+    Write-Verbose -Message "Retrieving outgoing email settings configuration "
+
+    $result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
+        $params = $args[0]
+        
+        $webApp = $null
```

Comment:
This $webApp = $null statement is redundant, if you just add -ErrorAction SilentlyContinue to the following line you will get $null out of that if the web app doesn't exist.